src/App.less

```diff
@@ -6,55 +6,55 @@ body {
 }
 
 .app-title {
-  font-family: "FF Oxide Solid";
-  font-style: normal;
-  font-weight: 300;
+  font-family   : "FF Oxide Solid";
+  font-style    : normal;
+  font-weight   : 300;
```

Comment:
nit: Is there a tool to format these so it's consistent and prevents churn?

Reply:
I was using `formatte` for .less but I figured out they don't have a CLI so I reformatted with Prettier.